build/bazel/remote/execution/v2/remote_execution.proto

```diff
@@ -539,7 +539,10 @@ message Command {
 
   // The arguments to the command. The first argument must be the path to the
   // executable, which must be either a relative path, in which case it is
-  // evaluated with respect to the input root, or an absolute path.
+  // evaluated with respect to the working directory, or an absolute path. Path
+  // separators native to the operating system running on the worker are used.
+  // Paths that do not contain contain any path separators must be resolved
+  // using the operating system's equivalent of the PATH environment variable.
```

Comment:
Are shell builtins or functions allowed here? Should we say anything about those possibilities?

(Useful posix reference: https://pubs.opengroup.org/onlinepubs/9699919799/utilities/V3_chap02.html#tag_18_09_01_01)